1-mandatory/2-people-i-know.js

```diff
@@ -403,6 +440,16 @@ This time, I only want the full names of the people who can multitask
 
 let colleaguesWhoCanMultitask = [];
 
+function getFriendsWhoMultitask(friend) {
+  for (let colleague of friend.colleagues) {
+    if (colleague.skills.includes("Multi-tasking")) {
+      colleaguesWhoCanMultitask.push(colleague.name);
```

Comment:
Issue: While this will work it's not good practice to modify something outside of your function within a function. Here `colleaguesWhoCanMultitask ` has been declared outside of the function so it's best not to change it unless we pass it into the function as a parameter.